Makefile

```diff
@@ -708,6 +714,8 @@ $(foreach E,$(ELIB_DIR), \
 $(eval $(call _import_lib,$(CONFIG_UK_BASE)/arch/$(UK_FAMILY)))
 # internal platform libraries
 $(eval $(call verbose_include,$(CONFIG_UK_BASE)/plat/Makefile.uk))
+# drivers
+$(eval $(call verbose_include,$(CONFIG_UK_BASE)/drivers/Makefile.uk))
```

Comment:
Maybe put this before platforms because afterwards we are including external platforms and I think the platforms should stay together.